pkg/analytics/analytics.go

```diff
@@ -290,10 +291,12 @@ func (a *AnalyticsImpl) Send() (*http.Response, error) {
 // * key : "value"
 // * key = "value"
 // * key = value
+// * key "value"
+
 func SanitizeValuesByKey(keysToFilter []string, replacementValue string, content []byte) ([]byte, error) {
 	for i := range keysToFilter {
 		filter := keysToFilter[i]
-		r, err := regexp.Compile("(?i)([\"']?\\w*" + filter + "\\w*\"?)(((\\s?[:]\\s?[\"'])[^\n\"']*([\"']))|((\\s?[=]\\s?[\"']?)[^\n\"']*([\"']?)))")
+		r, err := regexp.Compile("(?i)([\"']?\\w*" + filter + "\\w*\"?)(((\\s?[:]\\s?[\"'])[^\n\"']*([\"']))|((\\s?[= ,]\\s?[\"']?)[^\n\"']*([\"']?)))")
```

Comment:
what does the regexp change do?

Reply:
It supports different separators between key and value, before the only separator was `=` now we support the following pattern:
* key = value
* key , value
* key value